apps/src/applab/designElements/PropertyRow.jsx

```diff
@@ -16,7 +18,8 @@ var PropertyRow = React.createClass({
     lockState: React.PropTypes.oneOf([LockState.LOCKED, LockState.UNLOCKED, undefined]),
     isMultiLine: React.PropTypes.bool,
     handleChange: React.PropTypes.func,
-    handleLockChange: React.PropTypes.func
+    handleLockChange: React.PropTypes.func,
+    extraRowStyle: React.PropTypes.object
```

Comment:
This feels a little confusing to me. I wonder if we'd be better off either having a bool along the lines of isIdRow, or instead have this be something like styleOverride and pass in $.extend({}, rowStyle.container, idRowStyle). Either seem clearer to me than passing in an "extra" set of styles.